documentation/docs/steps/apiProviderDownload.md

```diff
@@ -0,0 +1,35 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
```

Comment:
Maybe [this documentation](https://github.com/SAP/jenkins-library/pull/3407/files#diff-ee7faaa35fc635d915b2b43d3ec58b24dafaae85c44bd43a88bd35c464330cd1) can be of interest to you.

Reply:
I checked the documentation created for APIKeyValueMapUpload and didn't see much differences which has to be changed. It looks similar to me.